docs/features/azure-iam/tutorials/azure-iam-aks.mdx

```diff
@@ -78,6 +79,10 @@ az aks get-credentials -n $AKS_CLUSTER_NAME -g $RESOURCE_GROUP
 To deploy Otterize, head over to [Otterize Cloud](https://app.otterize.com) and:
 
 1. Create a Kubernetes integration on the [Integrations page](https://app.otterize.com/integrations), and follow the instructions. *Make sure to enable enforcement mode for this tutorial.* If you already have a Kubernetes cluster connected, skip this step.
+    - Add the following value to your helm installtion, to enable Azure resource mapping with eBPF:
+        ```
+        helm upgrade ... --set networkMapper.nodeagent.enable=true
+        ```
```

Comment:
This should look like the info callout in the AWS tutorial (the `:::info ....` section)